packages/base/card-api.gts

```diff
@@ -1894,6 +1894,17 @@ export function subscribeToChanges(
     subscribers.set(fieldOrCard, changeSubscribers);
   }
   changeSubscribers.add(subscriber);
+
+  let fields = getFields(fieldOrCard, {
+    usedFieldsOnly: true,
+    includeComputeds: false,
+  });
+  Object.keys(fields).forEach((fieldName) => {
+    let value = peekAtField(fieldOrCard, fieldName);
+    if (isFieldDef(value)) {
```

Comment:
are we only subscribing to non-primitive field?? Or is this meant to be generic?? I think there is another function called `isCompoundField` if thats what you wanted to use

Reply:
Only compound field, I'll update it.

Reply:
Updated [fcaad22](https://github.com/cardstack/boxel/pull/832/commits/fcaad2297dcf3041224f116a7472fa3d6e8b74af)